packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -4187,14 +4632,54 @@ export class Quaternion {
         return this;
     }
 
+    public addToRef<T extends this>(other: DeepImmutable<T>, result: T): T {

```

Comment:
Missing docs for the new `Quaternion` methods.

Reply:
In 4e38dc4, I change the usage of "Tensor" in Tensor doc comments to "instance" (e.g. "Get a new Tensor set with ..." -> "Get a new instance set with..."). This means the `Tensor` doc comment works great as a doc comment for derived classes as well, including `Quaternion`.